chart/templates/secrets.yaml

```diff
@@ -62,3 +73,45 @@ metadata:
 type: Opaque
 stringData:
   PGRST_JWT_SECRET: {{ $pgrstJWTSecret | quote }}
+---
+{{- $pgConfig := dict "pguser" $user "pgpassword" $password "database" $database "pgHost" $postgresHost "pgPort" $postgresPort -}}
+
+{{- define "userlistConfig" -}}
+"{{ .pguser }}" "{{ .pgpassword }}"
+{{- end -}}
+
+{{- define "pgbouncerConfig" -}}
+[databases]
+{{ .database }} = host={{ .pgHost }} port={{ .pgPort }} dbname={{ .database }}
+
+[pgbouncer]
+listen_addr = 0.0.0.0
+listen_port = 5432
+unix_socket_dir =
+auth_user = {{ .pguser }}
+auth_file = /bitnami/pgbouncer/conf/userlist.txt
+auth_type = md5
+pool_mode = session
+max_client_conn = 100
```

Comment:
Can we make this a variable and then also add a global.db.connectionPooler.extraOptions